core/sockets.py

```diff
@@ -62,6 +62,9 @@ class SvSocketCommon:
     expanded: BoolProperty(default=False)
     custom_draw: StringProperty(description="For name of method which will draw socket UI (optionally)")
     prop_name: StringProperty(default='', description="For displaying node property in socket UI")
+    sv_is_linked: BoolProperty(description="Node is linked if (node.is_linked or node.sv_is_linked). "
```

Comment:
do you mean "socket" here?     `description="Socket is linked......"   `  ?

i do not know how I feel about this. I am not able to test extensively now.

Reply:
Yes socket. It looks like solution is better than using `if socket.links`. 

I will check one more opportunity later.

Reply:
no.. i mean is the description of `sv_is_linked`  correct?    `"Node is linked if (node.is_linked....."  there is no  `node.is_linked`  ?

Reply:
Yes, description is incorrect. Socket is linked if (`socket.is_linked` or socket.sv_is_linked`). I'll fix this later. 

If you don't like the idea with the `sv_is_linked` method I can offer another one more solution. 
The problem is with `dictionary in` node. When socket is being connected it is replaced by socket with type of `other` socket and links is recreated. But in this case `is_linked` attribute of the socket remains False until the end of update event and in process `sv_get` method returnes `NoDataError`. 

It is possible to create one more type of sockets (chameleon) which will include method `set_color` which will dye itself into color of other socket. In this case socket remains unchanged and its `is_linked` attribute returns correct status.